### PR TITLE
feat: create integration image based on infrastructure-bundle

### LINKED ISSUE #50: Remove circular dependency between infra bundle release
Problem:
After the release of the integration , the new version needs to be added to the infra bundle image , once the bundle is released the release of the manifest is needed in this repo. This is procedure is manual and often forgotten.

Solution proposals:
- Dispatch the manifest release workflow from infra bundle ci
- Create the ecs image based on the bundle image like k8s integration. (chosen)
   - Unify all code and context related to the integration in the same repo (nri-ecs)
   - Do a single step deployment ( pushing image and templates installation manifests )

### PR DESCRIPTION
**Pull request by gsanchezgavier**
This PR modifies the building Pipeline to generate an image based on the `newrelic/infrastructure-bundle` that adds the integration, same approach as nri-kubernetes.

It also publish the updated integration manifests on the same release workflow which Fixes #50

Next step is to remove nri-ecs from the `newrelic/infrastructure-bundle` image 

Summary:
- Creates a `newrelic/nri-ecs` image that replace the `newrelic/infrastructure-bundle`.
- Removes the `ENABLE_NRI_ECS` since the `nri-ecs` image runs the integration by default.
- Removes the `newrelic-nri-docker-config-fargate.yml` files since is already present in the `newrelic/infrastructure` because is taken from the `nri-docker` [repo](https://github.com/newrelic/nri-docker/blob/master/docker-config.yml).

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+on:
+  release:
+    types: [prereleased, released]
+
+jobs:
+  release-image-manifest:
+    name: Release image and upload manifests
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE_NAME: newrelic/nri-ecs
```
Comment (paologallinaharbur) [inline .github/workflows/release.yml:12]: have we created already the new repo giving the access to the fsiops?

**Review by paologallinaharbur — APPROVED** [review]
Everything looks good! 🚀

File: .github/workflows/release.yml (hunk 1 above)
Comment (gsanchezgavier) [reply .github/workflows/release.yml:12]: yes that was already in place

**Comment by gsanchezgavier** [comment]
just rebased from main after merging base branch. No changes added

**Pull request merged**